capa/features/common.py

```diff
@@ -344,7 +344,6 @@ def freeze_deserialize(cls, args):
 
 class Arch(Feature):
     def __init__(self, value: str, description=None):
-        assert value in VALID_ARCH
```

Comment:
if we remove this check then we should probably add checks within the rule parser to validate what a user specifies

Reply:
agreed, reason I removed them here is because our `arch` extraction fails for packed samples, e.g. via UPX

Reply:
see https://github.com/fireeye/capa/pull/770/files#diff-651412a054a596ef4cccaa6c9cd9f8632585de43dfd8012ff675ca19f935f600R55